src/libmongoc/tests/test-mongoc-client.c

```diff
@@ -2224,6 +2224,12 @@ test_client_buildinfo_hang (void)
 static void
 test_mongoc_client_change_openssl_ctx_before_ops (void)
 {
+   if (!test_framework_get_ssl ()) {
```

Comment:
Add `@ctest-skipped@@` somewhere in message, which CTest will scan for and consider the test skipped if it sees it.